### DISCUSSION TITLE
Bulk Import Posts via CSV

### DISCUSSION THREAD

**Discussion by Sunil-D-R** [GetPublii/Publii#1347] [General]
Is there any way to bulk import posts?
I imagine a tool or a plugin would be good

**Comment by candidexmedia** [comment]
What format are your posts in?

All of your posts, tags, and authors are saved in a `db.sqlite` database file. I'd recommend creating a site, creating a few posts with the different editors, and then opening the db file with a sqlite editor such as [SQLiteStudio](https://sqlitestudio.pl/) and [DB Browser for SQLite](https://sqlitebrowser.org/). Look at how the entries are structured, and format your csv that way for import.
